### PR TITLE
prompt-local-terminal 3.0.3 (new cask)

### PR DESCRIPTION
**Pull request by HuaDeity**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by krehel** [comment]
I'm afraid I don't understand what this is for.  This doesn't look like an application, and any application mentions for Prompt look to point to the App Store.

**Comment by HuaDeity** [comment]
@krehel 

This is a helper program that is necessary to open the local terminal in prompt3. In my case, it would be beneficial to use Homebrew for installation and upgrades.

File: Casks/p/prompt-local-terminal.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  sha256 "ce0acc6758f3d685a0918fb01fd5f5aea87ab90c42415f1b4cf57d79f4b98077"
+
+  url "https://download-cdn.panic.com/prompt/Install%20Prompt%20Local%20Terminal%20#{version}.zip"
+  name "Prompt Local Terminal"
+  desc "Helper to enable local terminal access in Prompt"
+  homepage "https://help.panic.com/prompt/prompt-local-terminal/"
+
+  livecheck do
+    url "https://download-cdn.panic.com/prompt/"
+    regex(%r{href="/prompt/Install%20Prompt%20Local%20Terminal%20(\d+\.\d+\.\d+)\.zip"}i)
```
Comment (bevanjkay) [inline Casks/p/prompt-local-terminal.rb:12]:
```suggestion
    regex(/href=.*?Install%20Prompt%20Local%20Terminal%20v?(\d+(?:\.\d+)+)\.zip/i)

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Comment by HuaDeity** [comment]
@bevanjkay 

I used the "createzap" command, but it only displayed "~/Library/LaunchAgents/com.panic.LocalTerminal.plist", which has already been added in launchctl.

**Comment by razvanazamfirei** [comment]
I'm sorry, @HuaDeity, but I agree with @krehel on this one! Thank you for submitting this and all your hard work, but we're going to pass at this point. 

Thank you!

**Pull request closed without merging**